<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>piRadio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bPlay">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bStop">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>0</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QListWidget" name="wChannels">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>128</y>
      <width>321</width>
      <height>113</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lPos">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>64</y>
      <width>56</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInfo">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>80</y>
      <width>56</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
